apis/betterself/v1/events/tests.py

```diff
@@ -3,6 +3,7 @@
 
 from apis.betterself.v1.tests.test_base import BaseAPIv1Tests, GetRequestsTestsMixin, PostRequestsTestsMixin
 from apis.betterself.v1.urls import API_V1_LIST_CREATE_URL
+from betterself.users.models import User
```

Comment:
maybe also lookup why cookiecutter decided to import user ... might save shooting yourself in the foot